packages/mobile-app/app/(tabs)/ui.tsx

```diff
@@ -20,13 +20,22 @@ export default function UiKit() {
       style={{
         gap: 10,
         padding: 10,
-        backgroundColor: scheme === "dark" ? "#333" : "#fff",
+        backgroundColor: scheme === "dark" ? "#101010" : "#fff",
       }}
     >
       <Text size="lg">Count: {count}</Text>
-      <Button title="Press me" onClick={() => setCount(count + 1)} />
-      <Box height="auto" bg="pink" borderWidth={2} borderColor="gray">
-        <Button title="Press me" onClick={() => setCount(count + 1)} />
+      <Button title="Press me - filled" onClick={() => setCount(count + 1)} />
+      <Box height="auto" bg="pink" borderWidth={2} borderColor="gray900">
+        <Button
+          styleVariant="outline"
```

Comment:
suggestion(non-blocking): I'd go with variant for brevity but up to you. Whatever we land on here, though, should be consistent across components.

Reply:
I'm cool with that 👍🏻